flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/RexNodeJsonDeserializer.java

```diff
@@ -334,6 +351,21 @@ private SqlOperator toOperator(JsonNode jsonNode, SerdeContext ctx) throws IOExc
             }
         }
 
+        // try to find operator from std operator table.
+        SqlStdOperatorTable.instance()
```

Comment:
does `ctx.getOperatorTable()` contain all operators used in Flink ?

Reply:
no, that is why I add this code, FlinkSqlOperatorTable dosen't not cover all of the operator, and I found that we use some operator like PATTERN_CONCAT directly.